examples/platform/silabs/MatterConfig.cpp

```diff
@@ -89,14 +89,6 @@ static chip::DeviceLayer::Internal::Efr32PsaOperationalKeystore gOperationalKeys
 #include <app/server/Server.h>
 
 #include <platform/silabs/platformAbstraction/SilabsPlatform.h>
-
```

Comment:
This PR has another change -- getting rid of the power manager workaround. Please mention it in the PR description (and possibly the title)